src/controller/helpers/zclFrameConverter.ts

```diff
@@ -1,19 +1,26 @@
-import {ZclFrame} from '../../zcl';
+import {ZclFrame, Utils as ZclUtils} from '../../zcl';
+import Debug from "debug";
 
 interface KeyValue {[s: string]: number | string}
 
-function attributeKeyValue(frame: ZclFrame): KeyValue {
+function attributeKeyValue(frame: ZclFrame, manufacturerID?: number): KeyValue {
     const payload: KeyValue = {};
+    let cluster = frame.Cluster;
```

Comment:
The if below can be removed, just use: `let cluster = ZclUtils.getCluster(frame.Cluster.ID, manufacturerID);` here